client/src/api/ProjectApiService.js

```diff
@@ -52,42 +52,28 @@ class ProjectApiService {
     console.log('THIS BASEPROJECT URL', this.baseProjectUrl);
 
     try {
-      const proj =  await fetch(this.baseProjectUrl, requestOptions);
-      const projectDetails = await proj.json()
-      return projectDetails._id
+      const proj = await fetch(this.baseProjectUrl, requestOptions);
+      const projectDetails = await proj.json();
+      return projectDetails._id;
```

Comment:
I hate to be a stickler about this stuff, but can you drop these changes? It makes it a lot harder to follow what is being changed the PR with a lot of other linting changes.

I know it's a hassle and we don't have a linter up and running for the repo yet, but adding the specifc section you're wanting to edit wth `git add --patch ${file}` or `git add -p ${file}` and picking the lines of your PR that aren't linting would make it a lot easier to process this.

There's a writeup in another comment on this PR review that has a walkthrough